eventOrganizer/Date.java

```diff
@@ -68,7 +109,4 @@ public static void main(String[] args) {
         System.out.println("Is invalidDate2 valid? " + invalidDate2.isValid()); // Should be false
         System.out.println("Is leapYearDate valid? " + leapYearDate.isValid()); // Should be true
     }
-};
```

Comment:
why is there a semicolon after this bracket? In java a semicolon usually only follows a {} if you are assigning an array
int a[] = {1, 2, 3};